xlint

```diff
@@ -284,16 +284,17 @@ for template; do
 	scan 'distfiles=.*downloads\.sourceforge\.net' 'use $SOURCEFORGE_SITE'
 	scan 'distfiles=.*savannah.nongnu\.org' 'use $NONGNU_SITE'
 	scan 'distfiles=.*archive\.ubuntu\.com' 'use $UBUNTU_SITE'
-	scan 'distfiles=.*xorg\.freedesktop\.org' 'use $XORG_SITE'
+	scan 'distfiles=.*x\.org/releases/individual' 'use $XORG_SITE'
 	scan 'distfiles=.*ftp.*debian\.org' 'use $DEBIAN_SITE'
 	scan 'distfiles=.*gnome\.org/pub' 'use $GNOME_SITE'
 	scan 'distfiles=.*www\.kernel\.org/pub/linux' 'use $KERNEL_SITE'
 	scan 'distfiles=.*cpan\.org/modules/by-module' 'use $CPAN_SITE'
-	scan 'distfiles=.*pypi\.python\.org' 'use $PYPI_SITE'
+	scan 'distfiles=.*files\.pythonhosted\.org/packages' 'use $PYPI_SITE'
 	scan 'distfiles=.*ftp\.mozilla\.org' 'use $MOZILLA_SITE'
 	scan 'distfiles=.*ftp\.gnu\.org/(pub/)?gnu' 'use $GNU_SITE'
 	scan 'distfiles=.*freedesktop\.org/software' 'use $FREEDESKTOP_SITE'
 	scan 'distfiles=.*download.kde.org/stable' 'use $KDE_SITE'
+	scan 'distfiles=.*xorg\.freedesktop\.org/wiki/' 'use XORG_HOME'
```

Comment:
I'm sure you mean to have a dollar-sign there.

Reply:
Ah, yes. Fixed.